docs/ai_actions/work_with_ai_actions.md

```diff
@@ -101,6 +101,43 @@ For a list of available settings, see [Edit existing AI actions](#edit-existing-
 
 1. Click **Save and close** to apply the changes or **Discard** to discard them and close the window.
 
+## Create AI actions that use [[= product_name_connect =]]
+
+If your organization uses [[= product_name_connect =]], you can build multi-step scenarios that define the logic needed to process your input data, for example, by merging the output of multiple AI services.
+One such example could be sending out a text for translation by one service, and then to another to make sure that the resulting translation is written in the right tone.
+
+!!! note "[[= product_name_connect =]] configuration required"
+
+    To use AI actions that interface with [[= product_name_connect =]], you must first [configure and initiate the connection]([[= developer_doc =]]/ai_actions/install_ai_actions/#configure-access-to-ibexa-connect), and [define templates](https://doc.ibexa.co/projects/connect/en/latest/scenarios/scenario_templates/#creating-templates) and/or [scenarios](https://doc.ibexa.co/projects/connect/en/latest/scenarios/creating_a_scenario/) in [[= product_name_connect =]].
+    
+1\. Navigate to the Admin Panel and select **AI actions**.
+
+2\. In the **AI actions** list, click **Create**.
+
+3\. In the slide-out pane, make choices like in [Create new AI actions](#create-new-ai-actions) but in the **Action handler** field, select the model that uses an [[= product_name_connect =]] scenario to process the request, for example `connect-image-to-text`, and then click **Create**.
+
+![[[= product_name_connect =]] handler](img/ai_action_connect_handler_selection.png)
+
+4\. In the **Global properties** area, set the required properties.
+
+5\. In the **Settings** area, select an existing scenario from a drop-down list.
+The list contains only those scenarios, which are compatible with the selected action type and are associated with your [[= product_name_connect =]] team.
```

Comment:
I think this point might be a bit misleading. For now, in this dropdown there are all scenarios visible that are assigned to a team. However, they might not necessarily be configured properly - meaning executing such an action will fail, therefore they are **not always** compatible.